### DISCUSSION TITLE
installation of rainbowkit for the dapp

### DISCUSSION THREAD

**Discussion by KABILESHTHEEMPEROR** [rainbow-me/rainbowkit#2120] [General]
npm init @rainbow-me/rainbowkit@latest i have installed this package. and how do i proceed next? there are only less guidance for the rainbowkit integration. someone help please

**Comment by 0xmkh** [comment]
@KABILESHTHEEMPEROR `npm init @rainbow-me/rainbowkit@latest` command just creates RainbowKit, wagmi and Next.js project for you. Of course i have no idea what project you're building, but i can suggest you to look at few of our features below and try to implement to your project.

- Theming. We support `lightTheme`, `darkTheme` and `midnightTheme`, but you can create your own theme as well https://www.rainbowkit.com/docs/theming
- Connect button. Import our custom created connect button component to your app https://www.rainbowkit.com/docs/connect-button
- Modal sizes. Choose what modal size you'd like to use https://www.rainbowkit.com/docs/modal-sizes

Is that helpful for you ?

**Reply by KABILESHTHEEMPEROR** [reply]
Is it available in javascript?

**Reply by 0xmkh** [reply]
We don't have vanilla javascript support. RainbowKit is React library.
